confluent_kafka/src/confluent_kafka.c

```diff
@@ -1187,6 +1187,59 @@ static void error_cb (rd_kafka_t *rk, int err, const char *reason, void *opaque)
 	CallState_resume(cs);
 }
 
+static void throttle_cb (rd_kafka_t *rk, const char *broker_name, int32_t broker_id,
+       int throttle_time_ms, void *opaque) {
+	Handle *h = opaque;
+	PyObject *ThrottleEvent_type, *throttle_event;
+	PyObject *result, *kwargs, *args;
+	CallState *cs;
+
+	cs = CallState_get(h);
+	if (!h->throttle_cb) {
+		/* No callback defined */
+		goto done;
+	}
+
+    ThrottleEvent_type = cfl_PyObject_lookup("confluent_kafka",
+                                              "ThrottleEvent");
+
+    if(!ThrottleEvent_type) {
+        /*ThrottleEvent class not found*/
+        goto err;
+    }
+
+    args = Py_BuildValue("(sii)", broker_name, broker_id, throttle_time_ms);
```

Comment:
I think it is customary to express time as float seconds in Python, thus a float formatter and 1000.0 * throttle_time_ms

Reply:
ack